bootstrap-salt.sh

```diff
@@ -1798,24 +1798,26 @@ __function_defined() {
 #                 process is finished so the script doesn't exit on a locked proc.
 #----------------------------------------------------------------------------------------------------------------------
 __wait_for_apt(){
-    echodebug "Checking if apt process is currently running."
-
     # Timeout set at 15 minutes
     WAIT_TIMEOUT=900
 
-    while ps -C apt,apt-get,aptitude,dpkg >/dev/null; do
-        sleep 1
-        WAIT_TIMEOUT=$((WAIT_TIMEOUT - 1))
+    # Set an initial exit code to 100; which matches the exit code of a locked apt process
+    (exit 100)
```

Comment:
I think, could we execute the passed in command first, instead of simulating "locked" exit code?
This way no need to wait an additional second for each `apt` call. Although it requires to save the exit code and return it later if not equals `100`.